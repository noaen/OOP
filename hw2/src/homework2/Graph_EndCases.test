# Sample Testing file

# Create a graph
CreateGraph graph1
CreateGraph graph2

# Create nodes
CreateNode n1 5
CreateNode n2 5
# Add nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge
AddEdge graph1 n1 n2

# getNodes
ListNodes graph1
ListNodes graph2

# getChildren
ListChildren graph1 n1
ListChildren graph1 n2

# Add an edge
AddEdge graph1 n1 n1
ListChildren graph1 n1
