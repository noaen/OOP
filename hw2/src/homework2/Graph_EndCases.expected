# Sample Testing file

# Create a graph
created graph graph1
created graph graph2

# Create nodes
created node n1 with cost 5
created node n2 with cost 5
# Add nodes
added node n1 to graph1
added node n2 to graph1

# Add an edge
added edge from n1 to n2 in graph1

# getNodes
graph1 contains: n1 n2
graph2 contains:

# getChildren
the children of n1 in graph1 are: n2
the children of n2 in graph1 are:

# Add an edge
added edge from n1 to n1 in graph1
the children of n1 in graph1 are: n1 n2
